LoRaEngine/README.md

```diff
@@ -32,34 +25,242 @@ The following guide describes the necessary steps to build and deploy the LoRaEn
 - SetUp an Azure IoT Hub instance and be familiar with [Azure IoT Edge module deployment](https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux) mechanism.
 - Be familiar with [Azure IoT Edge module development](https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux). Note: the following guide expects that your modules will be pushed to [Azure Container registry](https://azure.microsoft.com/en-us/services/container-registry/).
 
-### SetUp concentrator with Azure IoT Edge
+### SetUp Azure function facade and [Azure Container registry](https://azure.microsoft.com/en-us/services/container-registry/)
 
-- Note: if your LoRa chip set is connected by SPI bus please ensure that it is enabled, e.g. on [Raspberry Pi](https://www.raspberrypi.org/documentation/hardware/raspberrypi/spi/README.md).
-- Configure your `.env` file with your [Azure Container registry](https://azure.microsoft.com/en-us/services/container-registry/) access URL and credentials, e.g.:
+- TODO deploy them
+- Configure IoT Hub access key in the function:
+
+Copy `Connection string` with owner policy applied:
+
+![Copy IoT Hub Connection string](/pictures/CopyIoTHubString.PNG)
+
+Now paste it into `Application settings` -> `Collection strings` as `IoTHubConnectionString`:
+
+![Paste IoT Hub Connection string](/pictures/FunctionPasteString.PNG)
+
+- Extract Facade function `Host key` (needed in next step)
+
+![Extract Facade function Host key](/pictures/FunctionHostKey.PNG)
+
+- Configure your `.env` file with your [Azure Container registry](https://azure.microsoft.com/en-us/services/container-registry/) as well as the Facade access URL and credentials. Those variables will be used by our [Azure IoT Edge solution template](/LoRaEngine/deployment.template.json)
 
 ```{bash}
 CONTAINER_REGISTRY_USERNAME=myregistryrocks
 CONTAINER_REGISTRY_PASSWORD=ghjGD5jrK6667
 CONTAINER_REGISTRY_ADDRESS=myregistryrocks.azurecr.io
 FACADE_SERVER_URL=https://lorafacadefunctionrocks.azurewebsites.net/api/
-FACADE_AUTH_CODE=gkjhFGHFGGjhhg5645674==
+FACADE_AUTH_CODE=yourFunctionHostKey
 ```
 
+### SetUp concentrator with Azure IoT Edge
+
+- Note: if your LoRa chip set is connected by SPI bus please ensure that it is enabled, e.g. on [Raspberry Pi](https://www.raspberrypi.org/documentation/hardware/raspberrypi/spi/README.md).
```

Comment:
I personally preferred to link a more direct tutorial (https://www.makeuseof.com/tag/enable-spi-i2c-raspberry-pi/), we should probably harmonize this between doc file